packages/server/src/utils/server-hooks.ts

```diff
@@ -1,5 +1,5 @@
 export const ServerHooks = {
-  Login: 'Login',
+  ValidateLogin: 'ValidateLogin',
```

Comment:
This is a breaking change.. until we're ready to break the API, we can deprecate and support both names

Reply:
Yeah I agree will do the change